src/main/app/src/App.js

```diff
@@ -18,4 +20,17 @@ function App() {
   );
 }
 
-export default App;
+function AppRouter(){
+  return (
+    <Router on >
+      <Switch>
+       <Route exact path="/" component={App} />
+      <Route exact path="/signup" component={SignUp} />
```

Comment:
Autoformat this indent so that it line up with the `Route` element above, like so:

```javascript
<Switch>
  <Route exact path="/" component={App} />
  <Route exact path="/signup" component={SignUp} />
```